### ISSUE TITLE
OpenCollada Linux Release?

### ISSUE THREAD

**Issue by salamanderrake** [KhronosGroup/OpenCOLLADA#234]
Is there any way to make a release source package for opencollada? I just found out that the version of the library in the Gentoo Portage tree is going to be 2 years old soon and was hoping that there would be a source package release that we could add to the portage tree, from a central download repo.

**Comment by RemiArnaud** [comment]
old request, closing as probably not relevant anymore.
Please re-open if you still want this.

**Comment by hobbes1069** [comment]
I'm guessing this was closed en masse with all the other issues but official releases are still a good idea along with maintaining API/ABI compatibility within a series.

**Comment by RemiArnaud** [comment]
Yes, absolutely. Reopened. Thanks for feedback

**Comment by hasufell** [comment]
> I'm guessing this was closed en masse with all the other issues but official releases are still a good idea along with maintaining API/ABI compatibility within a series.

You are a dreamer ;)

It's now considered fancy to break API and ABI as often as possible and make people bundle your library. Google does it too, so it must be the right way!

**Comment by RemiArnaud** [comment]
that's because there is only ONE codebase .. https://www.youtube.com/watch?v=W71BTkUbdqE

**Comment by hasufell** [comment]
Yeah, facebook does it similarly wrong, that's why they had to introduce huge hacks to mercurial, because updating the repository alone would literally take hours.

**Comment by RemiArnaud** [comment]
ok, so we now have release numbers, and we do intend to follow http://semver.org/

But, it would be so much better if we had a tool that we would run on each PR with the build, that would check the compatibility so we make sure that we give the proper version number. 

This said, we do not intend to break compatibility, but will add APIs/ABIs as needed (that would be a minor version number update).

Does anyone has recommendations or would contribute to this effort ? It does not have to be cross platform, could be just on the mac or windows or linux build.
